demisto_sdk/commands/content_graph/objects/base_content.py

```diff
@@ -202,7 +202,11 @@ def _save(
         """
         for key, val in self.field_mapping.items():
             attr = getattr(self, key)
-            if key == "marketplaces":
+            if key == "docker_image":
```

Comment:
maybe we can check if the value is not primitive, and if this is the case convert it to str?